scavenger-agent-java/src/main/java/com/navercorp/scavenger/javaagent/ScavengerAgent.java

```diff
@@ -39,43 +39,18 @@ public static void premain(String args, Instrumentation inst) {
 
         new ScavengerBanner(config).printBanner(System.out);
 
-        CodeBase codeBase = null;
+        Scheduler scheduler = new Scheduler(config);
         if (!config.isAsyncCodeBaseScanMode()) {
-            codeBase = scanCodeBase(config);
+            boolean isScanSuccessful = scheduler.scanCodeBase();
```

Comment:
It's a small thing, but how about `scanSuccessful`?

Reply:
Fixed on d75c57c.